src/components/CallView/Grid/Grid.vue

```diff
@@ -523,6 +596,34 @@ export default {
 				this.currentPage = Math.max(0, this.numberOfPages - 1)
 			}
 		},
+
+		speakers(speakers) {
+			if (speakers.length === 0) {
+				return
+			}
+			speakers.forEach(speaker => {
+				this.promoteSpeaker(speaker)
+				if (this.unpromoteSpeakerTimer[speaker.attributes.nextcloudSessionId]) {
+					clearTimeout(this.unpromoteSpeakerTimer[speaker.attributes.nextcloudSessionId])
+				}
+			})
```

Comment:
`[ ].forEach()` does nothing
`clearTimeout(undefined)` won't throw an error

```suggestion
			speakers.forEach(speaker => {
				this.promoteSpeaker(speaker)
				clearTimeout(this.unpromoteSpeakerTimer[speaker.attributes.nextcloudSessionId])
			})
```

Reply:
And maybe keep consistent naming? I'm confused between `speaker` and `model`